src/CollapsibleTitledSection.lua

```diff
@@ -33,7 +33,7 @@ function CollapsibleTitledSectionClass.new(nameSuffix, titleText, showTitle, min
 	local self = {}
 	setmetatable(self, CollapsibleTitledSectionClass)
 
-	showTitle = showTitle or showTitle == nil
+	showTitle = true --showTitle or showTitle == nil -- if set to false this pretty much made this useless
```

Comment:
The logic of this is wrong: if a user passes in false we are overriding it and setting it to true.
We want it to default to true.
So I think we want something like
if (showTitle ~= false) then 
  showTitle = true
end

(if user doesn't pass in showTitle argument, what will showTitle evaluate to?  nil?  Does nil == false?)

Reply:
I was more referring to the fact that the showTitle value maybe doesn't actually do whatever its intended purpose is. If you set `showTitle = false` then the greyed out title bar does not show which kind of defeats the purpose of the CollapsibleTitledSectionClass no?

For example:

```Lua
local listFrame = VerticallyScalingListFrame.new("suffix")
local collapse = CollapsibleTitledSection.new("suffix", "titleText", false, true, false)
local multiChoice = LabeledMultiChoice.new("suffix", "labelText", choices, 1)

multiChoice:GetFrame().Parent = collapse:GetContentsFrame()
listFrame:AddChild(collapse:GetSectionFrame())
listFrame:AddBottomPadding()

listFrame:GetFrame().Parent = widgetGui
```
![robloxstudiobeta_2018-10-10_12-04-15](https://user-images.githubusercontent.com/6201941/46749853-9f96a700-cc84-11e8-8d63-69f963220b58.png)

As a complete side note, looking at how the terrain tool widget was designed I can see there would be some compatibility issues with the changes I proposed for the content frame auto sizing since they have their own approach to this problem. I think I should make the changes I proposed based on a parameter for compatibility purposes. Thoughts?

Reply:
OK.
So if showTitle param is useless let's just remove it.  Parties who are using the library will need to update their calls to this when they pull latest code, I'm OK with that.

Layout: sure, if you want to add a param to make that optional that's fine.  Please document clearly in-code what the param does, perhaps add explicit example.